<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Alarmz::MainWindow</class>
 <widget class="QMainWindow" name="Alarmz::MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Alarmz</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Alarmz.qrc">
    <normaloff>:/icons/Alarmz.ico</normaloff>:/icons/Alarmz.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="AlarmsTreeView" name="alarmsTreeView">
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="indentation">
       <number>0</number>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionOptions"/>
    <addaction name="separator"/>
    <addaction name="actionCloseWindow"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAlarms">
    <property name="title">
     <string>&amp;Alarms</string>
    </property>
    <addaction name="actionAdd"/>
    <addaction name="actionDuplicate"/>
    <addaction name="actionEdit"/>
    <addaction name="actionDelete"/>
    <addaction name="separator"/>
    <addaction name="actionEnable"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionHelpContents"/>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAlarms"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="windowTitle">
    <string>Main Toolbar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionAdd"/>
   <addaction name="actionEdit"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionHelpContents"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="Alarmz.qrc">
     <normaloff>:/silk/silk/page_white.png</normaloff>:/silk/silk/page_white.png</iconset>
   </property>
   <property name="text">
    <string>&amp;New...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="Alarmz.qrc">
     <normaloff>:/silk/silk/folder_page_white.png</normaloff>:/silk/silk/folder_page_white.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="Alarmz.qrc">
     <normaloff>:/silk/silk/disk.png</normaloff>:/silk/silk/disk.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Save &amp;As...</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionOptions">
   <property name="text">
    <string>O&amp;ptions...</string>
   </property>
  </action>
  <action name="actionCloseWindow">
   <property name="text">
    <string>&amp;Close Window</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionAdd">
   <property name="icon">
    <iconset resource="Alarmz.qrc">
     <normaloff>:/silk/silk/add.png</normaloff>:/silk/silk/add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Add...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionDuplicate">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;Duplicate</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionEdit">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Alarmz.qrc">
     <normaloff>:/silk/silk/pencil.png</normaloff>:/silk/silk/pencil.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Alarmz.qrc">
     <normaloff>:/silk/silk/delete.png</normaloff>:/silk/silk/delete.png</iconset>
   </property>
   <property name="text">
    <string>De&amp;lete</string>
   </property>
   <property name="shortcut">
    <string>Del</string>
   </property>
  </action>
  <action name="actionEnable">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>E&amp;nable</string>
   </property>
  </action>
  <action name="actionHelpContents">
   <property name="icon">
    <iconset resource="Alarmz.qrc">
     <normaloff>:/silk/silk/help.png</normaloff>:/silk/silk/help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Contents</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>&amp;About...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F1</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>AlarmsTreeView</class>
   <extends>QTreeView</extends>
   <header>AlarmsTreeView.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="Alarmz.qrc"/>
 </resources>
 <connections/>
</ui>
